### PR TITLE
<updated bots file and analytics tab changed to cloud>

### PR DESCRIPTION
**Pull request by asha-gadde**

### CODE REVIEW

File: Catalyst/source/bots.rst [hunk 1]
```diff
@@ -34,29 +34,43 @@ These BOTs are used to provide following services:
...
-* We can also schedule the bots.
+* We can also schedule the BOTs.
 
 
-**How to schedule the BOT**
+**How to schedule the BOTs**
 
-*Click on Bots tab.Click on any bot like "Terraform bot".Navigate to schedule tab.Enter the deatils like start date and end date  and choose minutes/hourly/daily/weekly/monthly and howmany times we want execute the bots.If yu want to execute the bot 
- with remote server enable the checkbox and If you want skip alternate runs in the execution enable the checkbox.Click on "schedule" button.
+* Select any BOT in BOTs library than navigate to schedule tab in the BOT.
```
Comment (RLIndia) [inline Catalyst/source/bots.rst]: Change "than" to "then"

File: Catalyst/source/bots.rst [hunk 2]
```diff
@@ -34,29 +34,43 @@ These BOTs are used to provide following services:
...
+
+* Enter the deatils like start date and end date  and choose minutes/hourly/daily/weekly/monthly and howmany times we want execute the bots.
```
Comment (RLIndia) [inline Catalyst/source/bots.rst]: Space between "how" and "many". Currently shows as one word.

Comment (RLIndia) [inline Catalyst/source/bots.rst]: Correct spelling of "details" which is currently spelled as "deatils"

Comment (RLIndia) [inline Catalyst/source/bots.rst]: Change "times we want execute the bots" to "times we want to execute the BOT"

File: Catalyst/source/index.rst [hunk 3]
```diff
@@ -28,11 +28,11 @@ The main documentation for the site is organized into a couple sections:
 
    gettingstarted
    menu-options
-   settings
```
Comment (RLIndia) [inline Catalyst/source/index.rst]: Is "settings" removed? Why?

File: Catalyst/source/index.rst [hunk 4]
```diff
@@ -28,11 +28,11 @@ The main documentation for the site is organized into a couple sections:
 
    gettingstarted
    menu-options
-   settings
+   workflow
    design
    workzone
    trackinwrk
```
Comment (RLIndia) [inline Catalyst/source/index.rst:34]: Should this be "track" rather than "trackinwrk"

File: Catalyst/source/menu-options.rst [hunk 5]
```diff
@@ -4,7 +4,7 @@ Introduction - RLCatalyst
 **RL Catalyst has four main menu options** -
 
 * :doc:`Workflow <Workflow>` : Create the Organization structure and manage DevOps assets across data center and cloud 
-* :doc:`Bots <bots> ` :  This Dashboard contains bots library and bots report and audit trail page.
+* :doc:`Bots <bots>`  :  This Dashboard contains bots library and bots report and audit trail page.
```
Comment (RLIndia) [inline Catalyst/source/menu-options.rst]: Always refer to "BOTs" rather than "bots"

**Review by RLIndia — CHANGES_REQUESTED** [review]

File: Catalyst/source/index.rst (hunk 3 above)
Comment (asha-gadde) [reply Catalyst/source/index.rst]: > Is "settings" removed? Why

In main dashboard data under settings tab called as "workflow".all org settings,devops roles all are under workflow tab that'y changed as workflow.

File: Catalyst/source/index.rst (hunk 4 above)
Comment (asha-gadde) [reply Catalyst/source/index.rst:34]: All files and doctress  are with that name only.If i rename that i need to change lot of settings and data.In catalyst usermanual  it shows as "track" only.

Comment (asha-gadde) [reply Catalyst/source/index.rst:34]: image

**Pull request merged**